app/src/test/java/com/kickstarter/viewmodels/CreatorDashboardHeaderHolderViewModelTest.java

```diff
@@ -0,0 +1,85 @@
+package com.kickstarter.viewmodels;
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.KSRobolectricTestCase;
+import com.kickstarter.factories.ProjectFactory;
+import com.kickstarter.factories.ProjectStatsFactory;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.RefTag;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.models.ProjectStats;
+
+import org.joda.time.DateTime;
+import org.junit.Test;
+import rx.observers.TestSubscriber;
+
+public class CreatorDashboardHeaderHolderViewModelTest extends KSRobolectricTestCase {
+  private CreatorDashboardHeaderHolderViewModel.ViewModel vm;
+
+  private final TestSubscriber<String> percentageFunded = new TestSubscriber<>();
+  private final TestSubscriber<String> projectBackersCountText = new TestSubscriber<>();
+  private final TestSubscriber<String> projectNameTextViewText = new TestSubscriber<>();
+  private final TestSubscriber<String> timeRemainingText = new TestSubscriber<>();
+  private final TestSubscriber<Pair<Project, RefTag>> startProjectActivity = new TestSubscriber<>();
+
+  protected void setUpEnvironment(final @NonNull Environment environment) {
+    this.vm = new CreatorDashboardHeaderHolderViewModel.ViewModel(environment);
+  }
+
+  @Test
+  public void testProjectBackersCountText() {
+    final Project project = ProjectFactory.project().toBuilder().backersCount(10).build();
+    final ProjectStats projectStats = ProjectStatsFactory.projectStats();
+    this.vm = new CreatorDashboardHeaderHolderViewModel.ViewModel(environment());
```

Comment:
rather than setting the vm in each test, let's call `setUpEnvironment(environment())` this instantiates the VM and also our test subscribers